.github/workflows/robot/internal/review/review.go

```diff
@@ -124,14 +125,18 @@ func (r *Assignments) Get(author string, docs bool, code bool) []string {
 
 	switch {
 	case docs && code:
+		log.Printf("Found docs and code changes.")
```

Comment:
Do you need to append `\n` for log.Printf or will it output it on a separate line anyway? I don't know for sure but there's also log.Println which is why I'm wondering.

Reply:
It appends a newline if one is missing.